packages/esm-patient-medications-app/src/medications-summary/medications-summary.component.tsx

```diff
@@ -16,51 +15,25 @@ export default function MedicationsSummary({ patient }: MedicationsSummaryProps)
   const launchAddDrugWorkspace =
     useLaunchWorkspaceRequiringVisit<AddDrugOrderWorkspaceAdditionalProps>('add-drug-order');
 
-  const {
-    data: allOrders,
-    error: error,
-    isLoading: isLoading,
-    isValidating: isValidating,
-  } = usePatientOrders(patient?.id, 'any');
-
-  const [pastOrders, activeOrders] = useMemo(() => {
-    const currentDate = new Date();
-    const pastOrders: Array<Order> = [];
-    const activeOrders: Array<Order> = [];
-
-    if (allOrders) {
-      for (let i = 0; i < allOrders.length; i++) {
-        const order = allOrders[i];
-        if (order.autoExpireDate && parseDate(order.autoExpireDate) < currentDate) {
-          pastOrders.push(order);
-        } else if (order.dateStopped && parseDate(order.dateStopped) < currentDate) {
-          pastOrders.push(order);
-        } else {
-          activeOrders.push(order);
-        }
-      }
-    }
-
-    return [pastOrders, activeOrders];
-  }, [allOrders]);
+  const { data: allOrders, error, isLoading, isValidating } = usePatientOrders(patient?.id);
 
   return (
     <div>
       <div style={{ marginBottom: '1.5rem' }}>
         {(() => {
           const displayText = t('activeMedicationsDisplayText', 'Active medications');
-          const headerTitle = t('activeMedicationsHeaderTitle', 'active medications');
+          const headerTitle = t('activeMedicationsHeaderTitle', 'Active medications');
 
           if (isLoading) return <DataTableSkeleton role="progressbar" />;
 
           if (error) return <ErrorState error={error} headerTitle={headerTitle} />;
 
-          if (activeOrders?.length) {
+          if (allOrders?.length) {
```

Comment:
Now that you've got the `activeOrders` from the `useActivePatientOrders` hook, shouldn't that be what gets used here and passed to `medications` below instead?